// ramda/src/index.js
module.exports = {};
module.exports.F = /*#__PURE__*/require("ramda/src/F.js");
module.exports.T = /*#__PURE__*/require("ramda/src/T.js");
module.exports.__ = /*#__PURE__*/require("ramda/src/__.js");
module.exports.add = /*#__PURE__*/require("ramda/src/add.js");
module.exports.addIndex = /*#__PURE__*/require("ramda/src/addIndex.js");
module.exports.addIndexRight = /*#__PURE__*/require("ramda/src/addIndexRight.js");
module.exports.adjust = /*#__PURE__*/require("ramda/src/adjust.js");
module.exports.all = /*#__PURE__*/require("ramda/src/all.js");
module.exports.allPass = /*#__PURE__*/require("ramda/src/allPass.js");
module.exports.always = /*#__PURE__*/require("ramda/src/always.js");
module.exports.and = /*#__PURE__*/require("ramda/src/and.js");
module.exports.any = /*#__PURE__*/require("ramda/src/any.js");
module.exports.anyPass = /*#__PURE__*/require("ramda/src/anyPass.js");
module.exports.ap = /*#__PURE__*/require("ramda/src/ap.js");
module.exports.aperture = /*#__PURE__*/require("ramda/src/aperture.js");
module.exports.append = /*#__PURE__*/require("ramda/src/append.js");
module.exports.apply = /*#__PURE__*/require("ramda/src/apply.js");
module.exports.applySpec = /*#__PURE__*/require("ramda/src/applySpec.js");
module.exports.applyTo = /*#__PURE__*/require("ramda/src/applyTo.js");
module.exports.ascend = /*#__PURE__*/require("ramda/src/ascend.js");
module.exports.ascendNatural = /*#__PURE__*/require("ramda/src/ascendNatural.js");
module.exports.assoc = /*#__PURE__*/require("ramda/src/assoc.js");
module.exports.assocPath = /*#__PURE__*/require("ramda/src/assocPath.js");
module.exports.binary = /*#__PURE__*/require("ramda/src/binary.js");
module.exports.bind = /*#__PURE__*/require("ramda/src/bind.js");
module.exports.both = /*#__PURE__*/require("ramda/src/both.js");
module.exports.call = /*#__PURE__*/require("ramda/src/call.js");
module.exports.chain = /*#__PURE__*/require("ramda/src/chain.js");
module.exports.clamp = /*#__PURE__*/require("ramda/src/clamp.js");
module.exports.clone = /*#__PURE__*/require("ramda/src/clone.js");
module.exports.collectBy = /*#__PURE__*/require("ramda/src/collectBy.js");
module.exports.comparator = /*#__PURE__*/require("ramda/src/comparator.js");
module.exports.complement = /*#__PURE__*/require("ramda/src/complement.js");
module.exports.compose = /*#__PURE__*/require("ramda/src/compose.js");
module.exports.composeWith = /*#__PURE__*/require("ramda/src/composeWith.js");
module.exports.concat = /*#__PURE__*/require("ramda/src/concat.js");
module.exports.cond = /*#__PURE__*/require("ramda/src/cond.js");
module.exports.construct = /*#__PURE__*/require("ramda/src/construct.js");
module.exports.constructN = /*#__PURE__*/require("ramda/src/constructN.js");
module.exports.converge = /*#__PURE__*/require("ramda/src/converge.js");
module.exports.count = /*#__PURE__*/require("ramda/src/count.js");
module.exports.countBy = /*#__PURE__*/require("ramda/src/countBy.js");
module.exports.curry = /*#__PURE__*/require("ramda/src/curry.js");
module.exports.curryN = /*#__PURE__*/require("ramda/src/curryN.js");
module.exports.dec = /*#__PURE__*/require("ramda/src/dec.js");
module.exports.defaultTo = /*#__PURE__*/require("ramda/src/defaultTo.js");
module.exports.descend = /*#__PURE__*/require("ramda/src/descend.js");
module.exports.descendNatural = /*#__PURE__*/require("ramda/src/descendNatural.js");
module.exports.difference = /*#__PURE__*/require("ramda/src/difference.js");
module.exports.differenceWith = /*#__PURE__*/require("ramda/src/differenceWith.js");
module.exports.dissoc = /*#__PURE__*/require("ramda/src/dissoc.js");
module.exports.dissocPath = /*#__PURE__*/require("ramda/src/dissocPath.js");
module.exports.divide = /*#__PURE__*/require("ramda/src/divide.js");
module.exports.drop = /*#__PURE__*/require("ramda/src/drop.js");
module.exports.dropLast = /*#__PURE__*/require("ramda/src/dropLast.js");
module.exports.dropLastWhile = /*#__PURE__*/require("ramda/src/dropLastWhile.js");
module.exports.dropRepeats = /*#__PURE__*/require("ramda/src/dropRepeats.js");
module.exports.dropRepeatsBy = /*#__PURE__*/require("ramda/src/dropRepeatsBy.js");
module.exports.dropRepeatsWith = /*#__PURE__*/require("ramda/src/dropRepeatsWith.js");
module.exports.dropWhile = /*#__PURE__*/require("ramda/src/dropWhile.js");
module.exports.either = /*#__PURE__*/require("ramda/src/either.js");
module.exports.empty = /*#__PURE__*/require("ramda/src/empty.js");
module.exports.endsWith = /*#__PURE__*/require("ramda/src/endsWith.js");
module.exports.eqBy = /*#__PURE__*/require("ramda/src/eqBy.js");
module.exports.eqProps = /*#__PURE__*/require("ramda/src/eqProps.js");
module.exports.equals = /*#__PURE__*/require("ramda/src/equals.js");
module.exports.evolve = /*#__PURE__*/require("ramda/src/evolve.js");
module.exports.filter = /*#__PURE__*/require("ramda/src/filter.js");
module.exports.find = /*#__PURE__*/require("ramda/src/find.js");
module.exports.findIndex = /*#__PURE__*/require("ramda/src/findIndex.js");
module.exports.findLast = /*#__PURE__*/require("ramda/src/findLast.js");
module.exports.findLastIndex = /*#__PURE__*/require("ramda/src/findLastIndex.js");
module.exports.flatten = /*#__PURE__*/require("ramda/src/flatten.js");
module.exports.flip = /*#__PURE__*/require("ramda/src/flip.js");
module.exports.flow = /*#__PURE__*/require("ramda/src/flow.js");
module.exports.forEach = /*#__PURE__*/require("ramda/src/forEach.js");
module.exports.forEachObjIndexed = /*#__PURE__*/require("ramda/src/forEachObjIndexed.js");
module.exports.fromPairs = /*#__PURE__*/require("ramda/src/fromPairs.js");
module.exports.groupBy = /*#__PURE__*/require("ramda/src/groupBy.js");
module.exports.groupWith = /*#__PURE__*/require("ramda/src/groupWith.js");
module.exports.gt = /*#__PURE__*/require("ramda/src/gt.js");
module.exports.gte = /*#__PURE__*/require("ramda/src/gte.js");
module.exports.has = /*#__PURE__*/require("ramda/src/has.js");
module.exports.hasIn = /*#__PURE__*/require("ramda/src/hasIn.js");
module.exports.hasPath = /*#__PURE__*/require("ramda/src/hasPath.js");
module.exports.head = /*#__PURE__*/require("ramda/src/head.js");
module.exports.identical = /*#__PURE__*/require("ramda/src/identical.js");
module.exports.identity = /*#__PURE__*/require("ramda/src/identity.js");
module.exports.ifElse = /*#__PURE__*/require("ramda/src/ifElse.js");
module.exports.inc = /*#__PURE__*/require("ramda/src/inc.js");
module.exports.includes = /*#__PURE__*/require("ramda/src/includes.js");
module.exports.indexBy = /*#__PURE__*/require("ramda/src/indexBy.js");
module.exports.indexOf = /*#__PURE__*/require("ramda/src/indexOf.js");
module.exports.init = /*#__PURE__*/require("ramda/src/init.js");
module.exports.innerJoin = /*#__PURE__*/require("ramda/src/innerJoin.js");
module.exports.insert = /*#__PURE__*/require("ramda/src/insert.js");
module.exports.insertAll = /*#__PURE__*/require("ramda/src/insertAll.js");
module.exports.intersection = /*#__PURE__*/require("ramda/src/intersection.js");
module.exports.intersperse = /*#__PURE__*/require("ramda/src/intersperse.js");
module.exports.into = /*#__PURE__*/require("ramda/src/into.js");
module.exports.invert = /*#__PURE__*/require("ramda/src/invert.js");
module.exports.invertObj = /*#__PURE__*/require("ramda/src/invertObj.js");
module.exports.invoker = /*#__PURE__*/require("ramda/src/invoker.js");
module.exports.is = /*#__PURE__*/require("ramda/src/is.js");
module.exports.isEmpty = /*#__PURE__*/require("ramda/src/isEmpty.js");
module.exports.isNil = /*#__PURE__*/require("ramda/src/isNil.js");
module.exports.isNotEmpty = /*#__PURE__*/require("ramda/src/isNotEmpty.js");
module.exports.isNotNil = /*#__PURE__*/require("ramda/src/isNotNil.js");
module.exports.join = /*#__PURE__*/require("ramda/src/join.js");
module.exports.juxt = /*#__PURE__*/require("ramda/src/juxt.js");
module.exports.keys = /*#__PURE__*/require("ramda/src/keys.js");
module.exports.keysIn = /*#__PURE__*/require("ramda/src/keysIn.js");
module.exports.last = /*#__PURE__*/require("ramda/src/last.js");
module.exports.lastIndexOf = /*#__PURE__*/require("ramda/src/lastIndexOf.js");
module.exports.length = /*#__PURE__*/require("ramda/src/length.js");
module.exports.lens = /*#__PURE__*/require("ramda/src/lens.js");
module.exports.lensIndex = /*#__PURE__*/require("ramda/src/lensIndex.js");
module.exports.lensPath = /*#__PURE__*/require("ramda/src/lensPath.js");
module.exports.lensProp = /*#__PURE__*/require("ramda/src/lensProp.js");
module.exports.lift = /*#__PURE__*/require("ramda/src/lift.js");
module.exports.liftN = /*#__PURE__*/require("ramda/src/liftN.js");
module.exports.lt = /*#__PURE__*/require("ramda/src/lt.js");
module.exports.lte = /*#__PURE__*/require("ramda/src/lte.js");
module.exports.map = /*#__PURE__*/require("ramda/src/map.js");
module.exports.mapAccum = /*#__PURE__*/require("ramda/src/mapAccum.js");
module.exports.mapAccumRight = /*#__PURE__*/require("ramda/src/mapAccumRight.js");
module.exports.mapObjIndexed = /*#__PURE__*/require("ramda/src/mapObjIndexed.js");
module.exports.match = /*#__PURE__*/require("ramda/src/match.js");
module.exports.mathMod = /*#__PURE__*/require("ramda/src/mathMod.js");
module.exports.max = /*#__PURE__*/require("ramda/src/max.js");
module.exports.maxBy = /*#__PURE__*/require("ramda/src/maxBy.js");
module.exports.mean = /*#__PURE__*/require("ramda/src/mean.js");
module.exports.median = /*#__PURE__*/require("ramda/src/median.js");
module.exports.memoizeWith = /*#__PURE__*/require("ramda/src/memoizeWith.js");
module.exports.mergeAll = /*#__PURE__*/require("ramda/src/mergeAll.js");
module.exports.mergeDeepLeft = /*#__PURE__*/require("ramda/src/mergeDeepLeft.js");
module.exports.mergeDeepRight = /*#__PURE__*/require("ramda/src/mergeDeepRight.js");
module.exports.mergeDeepWith = /*#__PURE__*/require("ramda/src/mergeDeepWith.js");
module.exports.mergeDeepWithKey = /*#__PURE__*/require("ramda/src/mergeDeepWithKey.js");
module.exports.mergeLeft = /*#__PURE__*/require("ramda/src/mergeLeft.js");
module.exports.mergeRight = /*#__PURE__*/require("ramda/src/mergeRight.js");
module.exports.mergeWith = /*#__PURE__*/require("ramda/src/mergeWith.js");
module.exports.mergeWithKey = /*#__PURE__*/require("ramda/src/mergeWithKey.js");
module.exports.min = /*#__PURE__*/require("ramda/src/min.js");
module.exports.minBy = /*#__PURE__*/require("ramda/src/minBy.js");
module.exports.modify = /*#__PURE__*/require("ramda/src/modify.js");
module.exports.modifyPath = /*#__PURE__*/require("ramda/src/modifyPath.js");
module.exports.modulo = /*#__PURE__*/require("ramda/src/modulo.js");
module.exports.move = /*#__PURE__*/require("ramda/src/move.js");
module.exports.multiply = /*#__PURE__*/require("ramda/src/multiply.js");
module.exports.nAry = /*#__PURE__*/require("ramda/src/nAry.js");
module.exports.partialObject = /*#__PURE__*/require("ramda/src/partialObject.js");
module.exports.negate = /*#__PURE__*/require("ramda/src/negate.js");
module.exports.none = /*#__PURE__*/require("ramda/src/none.js");
module.exports.not = /*#__PURE__*/require("ramda/src/not.js");
module.exports.nth = /*#__PURE__*/require("ramda/src/nth.js");
module.exports.nthArg = /*#__PURE__*/require("ramda/src/nthArg.js");
module.exports.o = /*#__PURE__*/require("ramda/src/o.js");
module.exports.objOf = /*#__PURE__*/require("ramda/src/objOf.js");
module.exports.of = /*#__PURE__*/require("ramda/src/of.js");
module.exports.omit = /*#__PURE__*/require("ramda/src/omit.js");
module.exports.on = /*#__PURE__*/require("ramda/src/on.js");
module.exports.once = /*#__PURE__*/require("ramda/src/once.js");
module.exports.or = /*#__PURE__*/require("ramda/src/or.js");
module.exports.otherwise = /*#__PURE__*/require("ramda/src/otherwise.js");
module.exports.over = /*#__PURE__*/require("ramda/src/over.js");
module.exports.pair = /*#__PURE__*/require("ramda/src/pair.js");
module.exports.partial = /*#__PURE__*/require("ramda/src/partial.js");
module.exports.partialRight = /*#__PURE__*/require("ramda/src/partialRight.js");
module.exports.partition = /*#__PURE__*/require("ramda/src/partition.js");
module.exports.path = /*#__PURE__*/require("ramda/src/path.js");
module.exports.paths = /*#__PURE__*/require("ramda/src/paths.js");
module.exports.pathEq = /*#__PURE__*/require("ramda/src/pathEq.js");
module.exports.pathOr = /*#__PURE__*/require("ramda/src/pathOr.js");
module.exports.pathSatisfies = /*#__PURE__*/require("ramda/src/pathSatisfies.js");
module.exports.pick = /*#__PURE__*/require("ramda/src/pick.js");
module.exports.pickAll = /*#__PURE__*/require("ramda/src/pickAll.js");
module.exports.pickBy = /*#__PURE__*/require("ramda/src/pickBy.js");
module.exports.pipe = /*#__PURE__*/require("ramda/src/pipe.js");
module.exports.pipeWith = /*#__PURE__*/require("ramda/src/pipeWith.js");
module.exports.pluck = /*#__PURE__*/require("ramda/src/pluck.js");
module.exports.prepend = /*#__PURE__*/require("ramda/src/prepend.js");
module.exports.product = /*#__PURE__*/require("ramda/src/product.js");
module.exports.project = /*#__PURE__*/require("ramda/src/project.js");
module.exports.promap = /*#__PURE__*/require("ramda/src/promap.js");
module.exports.prop = /*#__PURE__*/require("ramda/src/prop.js");
module.exports.propEq = /*#__PURE__*/require("ramda/src/propEq.js");
module.exports.propIs = /*#__PURE__*/require("ramda/src/propIs.js");
module.exports.propOr = /*#__PURE__*/require("ramda/src/propOr.js");
module.exports.propSatisfies = /*#__PURE__*/require("ramda/src/propSatisfies.js");
module.exports.props = /*#__PURE__*/require("ramda/src/props.js");
module.exports.range = /*#__PURE__*/require("ramda/src/range.js");
module.exports.reduce = /*#__PURE__*/require("ramda/src/reduce.js");
module.exports.reduceBy = /*#__PURE__*/require("ramda/src/reduceBy.js");
module.exports.reduceRight = /*#__PURE__*/require("ramda/src/reduceRight.js");
module.exports.reduceWhile = /*#__PURE__*/require("ramda/src/reduceWhile.js");
module.exports.reduced = /*#__PURE__*/require("ramda/src/reduced.js");
module.exports.reject = /*#__PURE__*/require("ramda/src/reject.js");
module.exports.remove = /*#__PURE__*/require("ramda/src/remove.js");
module.exports.repeat = /*#__PURE__*/require("ramda/src/repeat.js");
module.exports.replace = /*#__PURE__*/require("ramda/src/replace.js");
module.exports.reverse = /*#__PURE__*/require("ramda/src/reverse.js");
module.exports.scan = /*#__PURE__*/require("ramda/src/scan.js");
module.exports.sequence = /*#__PURE__*/require("ramda/src/sequence.js");
module.exports.set = /*#__PURE__*/require("ramda/src/set.js");
module.exports.slice = /*#__PURE__*/require("ramda/src/slice.js");
module.exports.sort = /*#__PURE__*/require("ramda/src/sort.js");
module.exports.sortBy = /*#__PURE__*/require("ramda/src/sortBy.js");
module.exports.sortWith = /*#__PURE__*/require("ramda/src/sortWith.js");
module.exports.split = /*#__PURE__*/require("ramda/src/split.js");
module.exports.splitAt = /*#__PURE__*/require("ramda/src/splitAt.js");
module.exports.splitEvery = /*#__PURE__*/require("ramda/src/splitEvery.js");
module.exports.splitWhen = /*#__PURE__*/require("ramda/src/splitWhen.js");
module.exports.splitWhenever = /*#__PURE__*/require("ramda/src/splitWhenever.js");
module.exports.startsWith = /*#__PURE__*/require("ramda/src/startsWith.js");
module.exports.subtract = /*#__PURE__*/require("ramda/src/subtract.js");
module.exports.sum = /*#__PURE__*/require("ramda/src/sum.js");
module.exports.swap = /*#__PURE__*/require("ramda/src/swap.js");
module.exports.symmetricDifference = /*#__PURE__*/require("ramda/src/symmetricDifference.js");
module.exports.symmetricDifferenceWith = /*#__PURE__*/require("ramda/src/symmetricDifferenceWith.js");
module.exports.tail = /*#__PURE__*/require("ramda/src/tail.js");
module.exports.take = /*#__PURE__*/require("ramda/src/take.js");
module.exports.takeLast = /*#__PURE__*/require("ramda/src/takeLast.js");
module.exports.takeLastWhile = /*#__PURE__*/require("ramda/src/takeLastWhile.js");
module.exports.takeWhile = /*#__PURE__*/require("ramda/src/takeWhile.js");
module.exports.tap = /*#__PURE__*/require("ramda/src/tap.js");
module.exports.test = /*#__PURE__*/require("ramda/src/test.js");
module.exports.andThen = /*#__PURE__*/require("ramda/src/andThen.js");
module.exports.times = /*#__PURE__*/require("ramda/src/times.js");
module.exports.toLower = /*#__PURE__*/require("ramda/src/toLower.js");
module.exports.toPairs = /*#__PURE__*/require("ramda/src/toPairs.js");
module.exports.toPairsIn = /*#__PURE__*/require("ramda/src/toPairsIn.js");
module.exports.toString = /*#__PURE__*/require("ramda/src/toString.js");
module.exports.toUpper = /*#__PURE__*/require("ramda/src/toUpper.js");
module.exports.transduce = /*#__PURE__*/require("ramda/src/transduce.js");
module.exports.transpose = /*#__PURE__*/require("ramda/src/transpose.js");
module.exports.traverse = /*#__PURE__*/require("ramda/src/traverse.js");
module.exports.trim = /*#__PURE__*/require("ramda/src/trim.js");
module.exports.tryCatch = /*#__PURE__*/require("ramda/src/tryCatch.js");
module.exports.type = /*#__PURE__*/require("ramda/src/type.js");
module.exports.unapply = /*#__PURE__*/require("ramda/src/unapply.js");
module.exports.unary = /*#__PURE__*/require("ramda/src/unary.js");
module.exports.uncurryN = /*#__PURE__*/require("ramda/src/uncurryN.js");
module.exports.unfold = /*#__PURE__*/require("ramda/src/unfold.js");
module.exports.union = /*#__PURE__*/require("ramda/src/union.js");
module.exports.unionWith = /*#__PURE__*/require("ramda/src/unionWith.js");
module.exports.uniq = /*#__PURE__*/require("ramda/src/uniq.js");
module.exports.uniqBy = /*#__PURE__*/require("ramda/src/uniqBy.js");
module.exports.uniqWith = /*#__PURE__*/require("ramda/src/uniqWith.js");
module.exports.unless = /*#__PURE__*/require("ramda/src/unless.js");
module.exports.unnest = /*#__PURE__*/require("ramda/src/unnest.js");
module.exports.until = /*#__PURE__*/require("ramda/src/until.js");
module.exports.unwind = /*#__PURE__*/require("ramda/src/unwind.js");
module.exports.update = /*#__PURE__*/require("ramda/src/update.js");
module.exports.useWith = /*#__PURE__*/require("ramda/src/useWith.js");
module.exports.values = /*#__PURE__*/require("ramda/src/values.js");
module.exports.valuesIn = /*#__PURE__*/require("ramda/src/valuesIn.js");
module.exports.view = /*#__PURE__*/require("ramda/src/view.js");
module.exports.when = /*#__PURE__*/require("ramda/src/when.js");
module.exports.where = /*#__PURE__*/require("ramda/src/where.js");
module.exports.whereAny = /*#__PURE__*/require("ramda/src/whereAny.js");
module.exports.whereEq = /*#__PURE__*/require("ramda/src/whereEq.js");
module.exports.without = /*#__PURE__*/require("ramda/src/without.js");
module.exports.xor = /*#__PURE__*/require("ramda/src/xor.js");
module.exports.xprod = /*#__PURE__*/require("ramda/src/xprod.js");
module.exports.zip = /*#__PURE__*/require("ramda/src/zip.js");
module.exports.zipObj = /*#__PURE__*/require("ramda/src/zipObj.js");
module.exports.zipWith = /*#__PURE__*/require("ramda/src/zipWith.js");
module.exports.thunkify = /*#__PURE__*/require("ramda/src/thunkify.js");
